martenbuild.cs

```diff
@@ -29,12 +29,18 @@ static void Main(string[] args)
                 RunNpm("install"));
 
             Target("mocha", DependsOn("install"), () =>
-                RunNpm("run test"));
-
-            Target("compile", DependsOn("clean"), () =>
-                Run("dotnet", $"build src/Marten.Testing/Marten.Testing.csproj --framework netcoreapp2.1 --configuration {configuration}"));
-
-            Target("test", DependsOn("compile"), () =>
+                RunNpm("run test"));
+
+            Target("compile", DependsOn("clean"), () =>
+                Run("dotnet", $"build src/Marten.Testing/Marten.Testing.csproj --framework netcoreapp2.1 --configuration {configuration}"));
+
+            Target("compile-noda-time", DependsOn("clean"), () =>
+                Run("dotnet", $"build src/Marten.NodaTime.Testing/Marten.NodaTime.Testing.csproj --framework netcoreapp2.1 --configuration {configuration}"));
+
+            Target("test-noda-time", DependsOn("compile-noda-time"), () =>
```

Comment:
@mysticmind I updated bullseye configuration. It would be great if you could check it :)

Reply:
@oskardudycz few suggestions:
- Remove target `test-noda-time` as a dependency from `test`. Note that adding `test-noda-time` as a dependency will run the nodatime extension tests first followed by primary tests
- Add `test-noda-time` as dependency to target `default` before `storyteller`

This will help us to see the chain of target runs clearly with primary tests followed by nodatime extension tests.